### DISCUSSION TITLE
GlobaLeaks & AWS Lightsail

### DISCUSSION THREAD

**Discussion by gianrolando** [globaleaks/globaleaks-whistleblowing-software#4291] [Q&A]
Hi.
After installing GlobaLeaks on AWS Lightsail with Ubuntu 24 I can't connect. When I connect to the public IP address of the instance I get "unable to reach the site".
What can I do? Is the installation incompatible?
Thanks.

**Comment by evilaliv3** [comment]
Hello @gianrolando 

Do you have a link to the site that you coud share?

**Reply by gianrolando** [reply]
Hello.
The instance has a public IP: http://3.121.185.161/

**Reply by evilaliv3** [reply]
At the moment i'm not able to reach anything. Is the server still up?
